crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -109,7 +112,14 @@ impl CallContractCommand {
 
 	fn display(&self) -> String {
 		let mut full_message = "pop call contract".to_string();
-		if let Some(path) = &self.path {
+		let path_flag = self.path.clone();
```

Comment:
This function is only to print the command in the screen to the user.
Is probably easier only:
```rs
if let Some(path) = &self.path {
  full_message.push_str(&format!(" --path {}", path.display()));
}
if let Some(path_pos) = &self.path_pos {
  full_message.push_str(&format!("{}", path_pos.display()));
}
```